src/pages/components/TabItem.qml

```diff
@@ -36,13 +32,9 @@ BackgroundItem {
 
```

Comment:
You can run browser like:

```
 QSG_VISUALIZE=overdraw sailfish-browser
```
Helps to spot extra blending. Green rectangles are good and red ones are bad.